docs/releases.md

```diff
@@ -1,23 +1,45 @@
 ## Releases
 
-## Preparation
-
-To **prepare the extension for a new version**, you must:
-
-- update and commit the version in `package.json`
-- update and commit the `CHANGELOG.md`
-- tag, push, and merge to the `master` branch on GitHub
-
-## Packaging
-
-To **build a signed extension .xpi**, you must: commit and push to the
-`production` branch (ideally as a merge commit from `master`) on GitHub.
-
-## Extension Signing
+### Instructions
+
+1. Update "version" in package.json (and package-lock.json)
```

Comment:
The "and package-lock.json" part concerns me; is this happening manually, or is it automatic? If it's manual, does it have to be? I haven't messed with our versioning stuff yet so I'm not really sure here...

Reply:
>The "and package-lock.json" part concerns me; is this happening manually, or is it automatic?

It'll automatically update at `install` (but that may touch a lot of other lines). Otherwise it's manual...

Reply:
I think if you set the version number using [`npm version`](https://docs.npmjs.com/cli/version) then it will update both the package.json and package-lock.json files automatically:

```sh
$ cat package.json | jq '.version' # "0.1.0-beta3"
$ cat package-lock.json | jq '.version' # "0.1.0-beta3"

$ npm version minor --no-git-tag-version # v0.1.0

$ cat package.json | jq '.version' # "0.1.0"
$ cat package-lock.json | jq '.version' # "0.1.0"
```